src/testing/builder/NewsModelBuilder.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import { LocalNewsModel } from '@integreat-app/integreat-api-client'
+import seedrandom from 'seedrandom'
+import { difference } from 'lodash'
+
+const MAX_PREDICTABLE_VALUE = 6
+const LANGUAGES = ['de', 'en', 'ar']
+
+class LocalNewsModelBuilder {
+  _newsCount: number
+  _seed: string
+  _city: string
+  _language: string
+
+  constructor (seed: string, newsCount: number, city: string, language: string) {
+    this._seed = seed
+    this._newsCount = newsCount
+    this._city = city
+    this._language = language
+  }
+
+  _predictableNumber (index: number, max: number = MAX_PREDICTABLE_VALUE): number {
+    return seedrandom(index + this._seed)() * max
+  }
```

Comment:
This isn't used at all, is it? If copying/pasting, you should take care that only necessary things are kept.